### ISSUE TITLE
Style/MixinGrouping autocorrects includes in reverse order

### ISSUE THREAD

**Issue by Drenmi** [bbatsov/rubocop#4059]
As pointed out by @PikachuEXE, the `Style/MixinGrouping` cop autocorrects:

```ruby
class Foo
  include Bar
  include Baz
end
```

into:

```ruby
class Foo
  include Bar, Baz
end
```

which reverses the ancestor order.

--------

## Expected behavior

RuboCop autocorrects to:

```ruby
class Foo
  include Baz, Bar
end
```

## Actual behavior

RuboCop autocorrects to:

```ruby
class Foo
  include Bar, Baz
end
```

## Steps to reproduce the problem

Run RuboCop on:

```ruby
class Foo
  include Bar
  include Baz
end
```

## RuboCop version

```
$ rubocop -V
0.47.1 (using Parser 2.3.3.1, running on ruby 2.3.3 x86_64-darwin15)
```

**Comment by Drenmi** [comment]
Well. I was too fast there. This cop doesn't have autocorrect yet. 😅

**Comment by PikachuEXE** [comment]
I guess having a note on the doc for `MixinGrouping` still helps
